rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/DefaultRocketMQListenerContainer.java

```diff
@@ -308,14 +314,14 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
     @Override
     public String toString() {
         return "DefaultRocketMQListenerContainer{" +
-            "consumerGroup='" + consumerGroup + '\'' +
-            ", nameServer='" + nameServer + '\'' +
-            ", topic='" + topic + '\'' +
-            ", consumeMode=" + consumeMode +
-            ", selectorType=" + selectorType +
-            ", selectorExpression='" + selectorExpression + '\'' +
-            ", messageModel=" + messageModel +
-            '}';
+                "consumerGroup='" + consumerGroup + '\'' +
```

Comment:
I found you changed the code style. Could you keep up with the code style on the [official website](http://rocketmq.apache.org/docs/code-guidelines/) ?

Reply:
I have make a new push

Reply:
I have made  a new push